plugins/linguist-backend/README.md

```diff
@@ -53,10 +53,25 @@ Here's how to get the backend up and running:
      apiRouter.use('/linguist', await linguist(linguistEnv));
    ```
 
-4. Now run `yarn start-backend` from the repo root
-5. Finally open `http://localhost:7007/api/linguist/health` in a browser and it should return `{"status":"ok"}`
+4. Optionally, if you would like to have the linguist backend supply language tags to your entities so they can be filtered on in the catalog index, setup the linguist tag processor in `packages/backend/src/plugins/catalog.ts`.
```

Comment:
I'd prefer having this as it's own section afterwards seeing as it's optional. This makes it easier for those already using Linguist to just jump to the new section and for those setting it up they can get that done and then add on this is they want it.